src/routes/dispensary/add/+page.svelte

```diff
@@ -2,18 +2,44 @@
   export let form;
 </script>
 
-<h1>Create Dispensary</h1>
+<h1>Add Dispensary</h1>
 <form method="POST" action="?/create">
   <label>
-    Name
+    Dispensary Name
     <input name="name" type="input" />
   </label>
+  <label>
+    Address
+    <input name="address" type="input" />
+  </label>
+  <label>
+    City 
+    <input name="city" type="input"/>
+  </label>
+  <label>
+    State
+    <input name="state" type="input"/>
+  </label>
+  <label>
+    Zip Code
+    <input name='zip' type="input" />
+  </label>
+  <label>
+    Status
+    <input name="status" type="input"/>
```

Comment:
I'm worried your Prettier is not working, I thought we had configured the indentation size to be 4 spaces and it should insert a space before the end of the self-closing tag: ` />`.  I'm also noticing this PR is reporting some merge conflicts, so I'm worried you are basing this work off the wrong commit.

Reply:
I thought I pulled down origin/main but I guess I didn't do something right.